great_expectations/execution_engine/pandas_execution_engine.py

```diff
@@ -420,6 +453,17 @@ class MetricDomainTypes.
 
         # Filtering if identity
         elif domain_type == MetricDomainTypes.IDENTITY:
+            if "ignore_row_if" in compute_domain_kwargs:
+                ignore_row_if = compute_domain_kwargs["ignore_row_if"]
+                if ignore_row_if == "all_values_are_missing":
+                    data = data.dropna(how="all")
+                elif ignore_row_if == "any_value_is_missing":
+                    data = data.dropna(how="any")
+                else:
+                    if ignore_row_if != "never":
+                        raise ValueError(
+                            f'Unrecognized value of ignore_row_if ("{ignore_row_if}").'
+                        )
```

Comment:
See comment in slack re: role of the IDENTITY-type MetricDomainTypes

Reply:
@jcampbell This open question has been captured in JIRA: `https://superconductive.atlassian.net/browse/GE-447`.  Thanks.